cmd/gardener-extension-provider-azure/app/app.go

```diff
@@ -202,8 +203,17 @@ func NewControllerManagerCommand(ctx context.Context) *cobra.Command {
 				return fmt.Errorf("could not update manager scheme: %w", err)
 			}
 
-			// add common meta types to schema for controller-runtime to use v1.ListOptions
-			metav1.AddToGroupVersion(scheme, machinev1alpha1.SchemeGroupVersion)
```

Comment:
Don't we still need this?

Reply:
I am actually a little bit confused about it but doesn't https://github.com/gardener/gardener-extension-provider-azure/blob/de1f4065099e7556d383cfe4787d162837ba2e12/cmd/gardener-extension-provider-azure/app/app.go#L198 already achieve the same ?

Reply:
That's true. Now, I'm not sure either. Maybe you can test it if it works this way, all the other extensions also can drop this lines then.

Reply:
I didn't see any issue so far in my tests but i will keep an eye